docs/source/admin/traffic_vault.rst

```diff
@@ -29,6 +29,28 @@ Configuring Traffic Vault
 =========================
 The following steps were taken to configure Riak in our environments.
 
+
+Self Signed Certificate configuration
+-------------------------------------
+Below shows how to create self signed certs for Trafficvault.
+
+Note: Self Signed Certificates are not recommended for production use. Intended for dev or learning purposes only. Modify subject as necessary.   
```

Comment:
Instead of `Note: `, could you use the appropriate admonition? There are `attention`, `caution`, `danger`, `error`, `hint`, `important`, `note`, `tip`, and `warning`. I'd suggest using `note` or `warning` based on the message, but feel free to choose whatever you think is appropriate - I can see an argument for several others and the exact choice isn't terribly important. You use them like this:
```rst
.. warning:: This text will appear inside a 'warning' admonition.

```

Reply:
Will do on both fronts! Thank you!